packages/front-end/components/Features/EditAttributesModal.tsx

```diff
@@ -131,6 +132,27 @@ export default function EditAttributesModal({ close }: { close: () => void }) {
                     ]}
                     sort={false}
                   />
+                  {form.watch(`attributeSchema.${i}.datatype`) === "string" && (
+                    <div className="my-1">
+                      <SelectField
+                        value={
+                          form.watch(`attributeSchema.${i}.format`) || "none"
+                        }
+                        onChange={(v) =>
+                          form.setValue(
+                            `attributeSchema.${i}.format`,
+                            v as SDKAttributeFormat
+                          )
+                        }
+                        style={{ width: 225 }}
+                        options={[
+                          { value: "none", label: "None" },
```

Comment:
Since we aren't showing labels for these dropdowns, `None` doesn't have context around it and it could be confusing to users what it refers to.  I'm thinking something like `Any Format`, `Freeform Text`, `Format: None`, etc. would convey the meaning better.  Open to other suggestions.

Long term when we redesign the attributes page this will no longer be an issue because we'll have plenty of space to add proper labels and help text to all of the input fields.  So this is more of a temporary solution.

Reply:
ok. I had just used the copy you provided in your suggestion but i've updated it to use your first suggestion c7dc4dc030e85b33047598e04d28cd201824b1b6